docs/diagnostics/QueryNestedFieldsByDot.md

```
@@ -9,23 +9,36 @@
-5. Разыменование ссылочных полей в секции "ГДЕ"
-`ГДЕ азКлиентаТовары.Ссылка.Дата МЕЖДУ &НачалоПериода И &КонецПериода`
+```bsl
+ИЗ
+	ВТ_РасчетыСКлиентами КАК ВТ_РасчетыСКлиентами
+		ЛЕВОЕ СОЕДИНЕНИЕ ВТ_ДанныеЗаказовКлиента КАК ВТ_ДанныеЗаказовКлиента
+		ПО ВТ_РасчетыСКлиентами.АналитикаУчетаПоПартнерам.Партнер = ВТ_ДанныеЗаказовКлиента.Партнер
+```
+4. Разыменование ссылочных полей в виртуальных таблицах
+```bsl
+ИЗ
+	РегистрНакопления.РасчетыСКлиентами.Обороты(
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)